clang/test/Parser/objc-prefixes2.m

```diff
@@ -0,0 +1,18 @@
+// RUN: %clang_cc1 %s -Wobjc-prefixes=NS,NSCF,NSURL -fsyntax-only -verify
```

Comment:
is it worth adding a `-Wobjc-forbidden-prefixes=NS`, so you can e.g. forbid things outside of Foundation from defining things with the `NS` prefix?

Reply:
Hmmm, maybe, yes. Not hard to add.